src/data/community-events.ts

```diff
@@ -0,0 +1,330 @@
+export interface CommunityEvent {
+  title: string
+  to: string
+  location: string
+  description: string
+  startDate: string
+  endDate: string
+}
```

Comment:
Per our conventions, would refactor to `type`, move to/export from `@/lib/types`, and then import it here and in `UpcomingEventsList.tsx`